src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DayRent = 40;
+  let totalRent = 0;
+
+  totalRent = DayRent * days;
+
+  if (days >= 7) {
+    totalRent = DayRent * days - 50;
```

Comment:
Prefer using `if` with `return` to simplify the logic and avoid the need for an `else` block.